GNUmakefile

```
@@ -828,6 +828,10 @@ website-misspell: ## [CI] Website Checks / misspell
 	@echo "make: Website Checks / misspell..."
 	@misspell -error -source text website/
 
+website-placeholder-check: ## [CI] Website Checks / website-placeholder-check
```

```suggestion
website-placeholder-check: prereq-go ## [CI] Website Checks / website-placeholder-check
```

This makes sure that the Go version is installed so it will work locally or in a GHA.